knox-webshell-ui/webshell-ui/assets/.gitkeep

```diff
@@ -0,0 +1,17 @@
+##########################################################################
```

Comment:
What's the purpose of this file/folder? If there are no assets now, we should not add the folder and remove all references.